README.md

```diff
@@ -0,0 +1,73 @@
+<div align="center">
+  <img src="./docs/mmcv-logo.png" width="300"/>
+</div>
+
+
+[![PyPI](https://img.shields.io/pypi/v/mmcv)](https://pypi.org/project/mmcv) [![badge](https://github.com/open-mmlab/mmcv/workflows/build/badge.svg)](https://github.com/open-mmlab/mmcv/actions) [![codecov](https://codecov.io/gh/open-mmlab/mmcv/branch/master/graph/badge.svg)](https://codecov.io/gh/open-mmlab/mmcv) [![license](https://img.shields.io/github/license/open-mmlab/mmcv.svg)](https://github.com/open-mmlab/mmcv/blob/master/LICENSE)
+
+
+## Introduction
+
+MMCV is a foundational python library for computer vision research and supports many
+research projects in MMLAB, such as [MMDetection](https://github.com/open-mmlab/mmdetection) and [MMAction](https://github.com/open-mmlab/mmaction).
+
+It provides the following functionalities.
+
+- Universal IO APIs
+- Image/Video processing
+- Image and annotation visualization
+- Useful utilities (progress bar, timer, ...)
+- PyTorch runner with hooking mechanism
+- Various CNN architectures
+- High-quality implementation of common CUDA ops
+
+See the [documentation](http://mmcv.readthedocs.io/en/latest) for more features and usage.
+
+Note: MMCV requires Python 3.6+.
+
+
+## Installation
+
+Try and start with
+
+```bash
+pip install mmcv
+```
+
+or install from source
+
+```bash
+git clone https://github.com/open-mmlab/mmcv.git
+cd mmcv
+pip install -e .
+```
+
+If you are on macOS, replace the last command with
+
+```bash
+CC=lang CXX=clang++ CFLAGS='-stdlib=libc++' pip install -e .
+```
+
+We also provide pre-build mmcv with corresponding Pytorch and CUDA versions
```

Comment:
Note that the building process may take up to 10 mins. We also provide pre-built mmcv with corresponding Pytorch and CUDA versions to simplify the building.